src/main/java/com/gamingmesh/jobs/config/ConfigManager.java

```diff
@@ -1148,6 +1148,7 @@ else if (commandSection.isList("command"))
 	    job.setBossbar(bossbar);
 	    job.setRejoinCd(rejoinCd);
 	    job.setMaxLevelCommands(jobSection.getStringList("commands-on-max-level"));
+            job.setIgnoreMaxJobs(jobSection.getBoolean("ignore-jobs-max", false));

```

Comment:
Follow the indentiation of the current code (instead of TAB, use spaces). For `getBoolean` you don't need to give `false` parameter, because its returns as false by default if the config property not exist.